lib.js

```diff
@@ -111,6 +133,16 @@ module.exports = (options) => {
                 return !pagesDomainsNames.includes(domain);
             });
 
+            const needsNoRenewal =
+                !options.forceRenewal &&
+                domainsToCreate.length === 0 &&
+                pagesDomains.every(hasValidCertificate);
+
+            if (needsNoRenewal) {
+                console.log(`All domains (${options.domain.join(', ')}) have a valid certificate (expiration in more than ${EXPIRATION_IN_DAYS} days)`);
+                process.exit(0);
```

Comment:
I would try to avoid this `process.exit` here - this file should only be used as a library for the main program (main.js). I think the better approach would be to refactor this file so the exported function (`getCertificates`) resolves successfully but with an empty array as a result, which means that everything went OK but no new certificates needed to be generated.